bliss/sle/bin/examples/rcf_api_test.py

```diff
@@ -52,7 +52,11 @@
 rcf_mngr = bliss.sle.RCF(
     hostname='atb-ocio-sspsim.jpl.nasa.gov',
     port=5100,
-    inst_id='sagr=LSE-SSC.spack=Test.rsl-fg=1.rcf=onlc2'
+    inst_id='sagr=LSE-SSC.spack=Test.rsl-fg=1.rcf=onlc2',
+    spacecraft_id=250,
+    trans_frame_ver_num=1,
+    auth_level="bind",
+    peer_auth_level="bind"
```

Comment:
We shouldn't need both of these. The auth level should cover outgoing/inbound auth in all 3 cases. I.e., 'Bind' level auth means the outgoing bind request is auth'ed and the receipt has auth info from the harness that we should decode